### DISCUSSION TITLE
Using GitHub Copilot Business Plan Across Multiple Organizations

### DISCUSSION THREAD

**Discussion by katagiri-nami** [community/community#144636] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

If a user belongs to multiple organizations on GitHub and one of those organizations subscribes to the GitHub Copilot Business plan, will the user be able to use Copilot features for private repositories in the other organization as well?

Additionally, if a user has an individual GitHub Copilot subscription, is it correct to assume that they can use Copilot features for private repositories in the organizations they belong to?

**Comment by nicolaslj** [comment]
Same question here ! :)

**Comment by atianpup** [comment, marked as answer]
Hey there! 😊

If you’re part of multiple organisations on GitHub and one has a GitHub Copilot Business plan, Copilot features will only work for repositories in that organization. They don’t extend to private repos in other organisations unless those organisations have their own subscription.

Yes, with an individual GitHub Copilot subscription, you can use Copilot for private repos in organisations. Restrictions on Copilot usage are typically only available for organizations with Enterprise plans, so in most cases, you should be able to use it freely.

Hope this clears things up!

**Reply by katagiri-nami** [reply]
Got it, understood! Thank you so much for the clarification! 😊

**Reply by PanTesla** [reply]
This is outdated now, right?
